pendulum/tz/timezone.py

```diff
@@ -58,6 +58,9 @@ def __new__(cls, key: str) -> Timezone:
         except zoneinfo.ZoneInfoNotFoundError:
             raise InvalidTimezone(key)
 
+    def __eq__(self, other: Any) -> bool:
```

Comment:
```suggestion
    def __eq__(self, other: object) -> bool:
```

Reply:
What's the difference in this case?

Reply:
Consistency. We already use `object` in other dunder methods and IIRC it's recommended.